uaclient/apt.py

```diff
@@ -75,7 +75,16 @@ def add_auth_apt_repo(repo_filename: str, repo_url: str, credentials: str,
 
     # Does this system have updates suite enabled?
     policy, _err = util.subp(['apt-cache', 'policy'])
-    updates_enabled = bool(' %s-updates/' % series in policy)
+    updates_enabled = False
+    for line in policy.splitlines():
+        # We only care about $suite-updates lines
+        if 'a={}-updates'.format(series) not in line:
```

Comment:
Can we combine these two conditionals

# We only care about $suite-updates from the Ubuntu archive
if 'o=Ubuntu,a={}-updates'.format(series) not in line:
    updates_enabled = True
    break

Reply:
I considered this, but I'm not 100% sure that `o=` and `a=` will always be next to one another in that order, so I went with a more forgiving approach.